From: Joe Carnuccio <joe.carnuccio@qlogic.com>
Date: Fri, 11 Apr 2014 16:54:18 -0400
Subject: qla2xxx: Correction to ISP27xx template entry types 256 and 258.
References: bnc#879633
Patch-Mainline: v3.17
Git-commit: 7095388f7eb2ff9fcd2ca17248a73de555f4d09b

The window address inserted into the dump should be have been
incremented by 1 rather than 4.

Signed-off-by: Joe Carnuccio <joe.carnuccio@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/qla2xxx/qla_tmpl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/scsi/qla2xxx/qla_tmpl.c b/drivers/scsi/qla2xxx/qla_tmpl.c
index 2909804..8469405 100644
--- a/drivers/scsi/qla2xxx/qla_tmpl.c
+++ b/drivers/scsi/qla2xxx/qla_tmpl.c
@@ -249,7 +249,7 @@ qla27xx_read_window(__iomem struct device_reg_24xx *reg,
 		qla27xx_insert32(base, buf, len);
 		readn(window, buf, len);
 		window += width;
-		base += width;
+		base++;
 	}
 }
 
-- 
1.7.12.4

